inttest/footloose-alpine/Dockerfile

```diff
@@ -25,7 +25,7 @@ ENV KUBECONFIG=/var/lib/k0s/pki/admin.conf
 
 # Install etcd for smoke tests with external etcd
 RUN wget https://github.com/etcd-io/etcd/releases/download/v3.5.1/etcd-v3.5.1-linux-$ETCD_ARCH.tar.gz
-RUN tar -xvf etcd-v3.5.1-linux-$ETCD_ARCH.tar.gz -C /opt --strip-components=1
+RUN tar -xvf etcd-v3.5.2-linux-$ETCD_ARCH.tar.gz -C /opt --strip-components=1
```

Comment:
Version mismatch between this and the above `RUN` command breaks all smokes 😄 

Could we get this somehow from the `Makefile.variables` so we'd have only one place to get the version?

Reply:
that was done in 0bc4dcfc6515c4944ed6a8400538a55b23fa5872